docs/table_design/Data_distribution.md

```diff
@@ -4,7 +4,8 @@ When you create a table, you must specify the data distribution method by config
 
 > NOTICE
 >
-> Since v2.5.7, StarRocks can automatically set the number of buckets (BUCKETS) when you create a table or add a partition. You no longer need to manually set the number of buckets. However, if the performance does not meet your expectations after StarRocks automatically sets the number of buckets and you are familiar with the bucketing mechanism, you can still [manually set the number of buckets](#determine-the-number-of-buckets).
+> - Since v3.1, StarRocks supports random bucketing, which which randomly distribute data across all buckets. You do not need to specify a bucketing key when creating a table or adding partitions, making the table creation statement more user-friendly. For more information, [Random bucketing](#random-bucketing-since-v31).
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/f832a562-cc95-4e32-8a4c-9d512bbbb60e)
should be "distributes"